Makefile

```diff
@@ -33,7 +33,6 @@ CLI_PLATFORMS ?= linux/amd64,linux/arm64,darwin/amd64,darwin/arm64
 VIZ_PLATFORMS ?= linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
 DOCKER_BUILDX_CMD ?= docker buildx
 IMAGE_BUILD_CMD ?= $(DOCKER_BUILDX_CMD) build
-IMAGE_BUILD_EXTRA_OPTS ?=
```

Comment:
@mbobrovskyi  Could we restore this variable?
By this PR, we drop the capabilities to specify additional building commands like `--no-cache` and `--progress=plain` which allows us to debug container buildings.

@mimowo Do you have any reason to drop this extra command?

Reply:
Ah, I see. Let me restore this variable.

Reply:
> @mimowo Do you have any reason to drop this extra command?

Ah I didn't see the use-case of passing `--no-cache`, sure, let's restore it.

Reply:
Done in #5568